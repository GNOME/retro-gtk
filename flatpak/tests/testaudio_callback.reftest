[Retro Reftest]
Path=/testaudio_callback
Core=/app/lib/libretro/testaudio_callback_libretro.so

[Options]

[Frame 0]
State=Refresh
Video=testaudio_callback.png
