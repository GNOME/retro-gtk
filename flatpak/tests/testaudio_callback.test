[Retro Reftest]
Path=/testaudio_callback
Core=/app/lib/libretro/testaudio_callback_libretro.so

[Frame 0]
Tests=Run;Video;StateRefresh;
Video=testaudio_callback.png
