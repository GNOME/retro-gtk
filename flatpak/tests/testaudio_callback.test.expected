/testaudio_callback/Boot: OK
/testaudio_callback/0/Run: OK
/testaudio_callback/0/Video: OK
/testaudio_callback/0/StateRefresh: OK
